icons/box-select-pointer.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1,.cls-2{fill:none;stroke:#000;stroke-miterlimit:10;stroke-width:2px;}.cls-1{stroke-linecap:round;}</style></defs><rect class="cls-1" x="2" y="2" width="4" height="4" rx="1"/><rect class="cls-1" x="18" y="2" width="4" height="4" rx="1"/><rect class="cls-1" x="2" y="18" width="4" height="4" rx="1"/><line class="cls-1" x1="4" y1="9" x2="4" y2="15"/><line class="cls-1" x1="20" y1="9" x2="20" y2="11"/><line class="cls-1" x1="15" y1="4" x2="9" y2="4"/><line class="cls-1" x1="11" y1="20" x2="9" y2="20"/><path class="cls-2" d="M14.33,22h-.09a.35.35,0,0,1-.24-.32v-10a.34.34,0,0,1,.33-.34.32.32,0,0,1,.21.08l7.34,6a.33.33,0,0,1-.21.59H17.18l-2.57,3.85A.35.35,0,0,1,14.33,22Z"/></svg>
```

Comment:
using the same cursor from the suggested lasso+cursor

to get 2 pixels spacing the bottom line becomes a circle, looks wrong to me, but I'm not sure, so here's that for comparison

![Screen Shot 2020-09-08 at 8 52 11 AM](https://user-images.githubusercontent.com/41239783/92478732-98e67680-f1b0-11ea-90ab-94f97dd0db07.png)
```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <rect rx="1" height="4" width="4" y="2" x="2" />
  <rect rx="1" height="4" width="4" y="2" x="18" />
  <rect rx="1" height="4" width="4" y="18" x="2" />
  <line y2="14" x2="4" y1="10" x1="4" />
  <line y2="12" x2="20" y1="10" x1="20" />
  <line y2="4" x2="10" y1="4" x1="14" />
  <line y2="20" x2="10" y1="20" x1="10" />
  <path d="M14.33 22h-.09a.35.35 0 01-.24-.32v-10a.34.34 0 01.33-.34c.08 0 .15.03.21.08l7.34 6a.33.33 0 01-.21.59h-4.49l-2.57 3.85a.35.35 0 01-.28.14v0z" />
</svg>
```

Reply:
Thanks for all of the info. Sorry for requiring all the hand holding. If I could make a suggestion here - I don't think the box select looks very good broken apart like that. I need to learn more about how to output clean svg code as clearly Illustrator doesn't handle it very well....but I would like to interject that since the box and anchor point corners are technically not separate elements AND it looks not so cohesive with 2px spacing that we change to something more like this: 
![image](https://user-images.githubusercontent.com/60020191/92668499-9e3ce180-f2d4-11ea-9784-cbbe0f6ee6fb.png)

Reply:
Can you both post an 100% scale version of it, maybe that better to judge

Reply:
![box select cursor 2-100](https://user-images.githubusercontent.com/60020191/92967871-9baacb00-f43f-11ea-9184-1a26c2f73266.jpg)

Reply:
Hmm I don't know. I don't think I will refer this to a "Select box". I think the one with the gaps look more like you can select something.

This icon can also present an square draw icon.

Btw we can also go more this way? Feel free to post more of your work in our discord channel!
![image](https://user-images.githubusercontent.com/11825403/93124252-9eb8ec00-f6c9-11ea-910d-d5534e082ce1.png)